pkg/jira/create.go

```diff
@@ -230,6 +235,11 @@ func constructCustomFields(fields map[string]string, data *createRequest) {
 				data.Fields.M.customFields[configured.Key] = val
 			}
 		}
+
+		if !found {
+			fmt.Fprintf(os.Stderr, "\nInvalid custom field specified: %s\n", key)
+			os.Exit(1)
```

Comment:
Since this is a package, instead of exiting we should return appropriate error and let the caller decide what to do with the error.

Reply:
Fixed.